README/Data/Trie/NonDependent.agda

```diff
@@ -148,33 +167,38 @@ module Lexer
 -- A small set of keywords for a language with expressions of the form
 -- `let x = e in b`.
 
-data TOK : Set where
-  LET EQ IN : TOK
-  LPAR RPAR : TOK
-  ID : String → TOK
 
-toks : List⁺ (String × TOK)
-toks = ("let" , LET)
-     ∷ ("="   , EQ)
-     ∷ ("in"  , IN)
-     ∷ []
+letIn : Lexer 0ℓ
+letIn = record { LetIn } module LetIn where
 
```

Comment:
Fascinating. This is valid syntax in 2.6.1 but in 2.6.0.1 the
named where-clause `LetIn` is not in scope for the RHS.